pkg/disk/type_bsd.go

```diff
@@ -22,3 +22,11 @@ package disk
 func getFSType(fstype [16]int8) string {
 	return b2s(fstype[:])
 }
+
+func b2s(bs []int8) string {
```

Comment:
There is no reason to keep this it as separate function.  This code needs to be inside `getFSType()`.  Sorry for the late catch.